Week2/homework/maartjes_work.js

```diff
@@ -44,4 +44,12 @@ const tuesday = [
 
 const tasks = monday.concat(tuesday);
 
+const timeInHours = tasks.map((minutes) => minutes.duration / 60);
+
+const maartjeWorkTime = timeInHours.filter((duration) => duration >= 2);
+const rate = 15;
+const maartjeEarnPerHour = maartjeWorkTime.map((duration) => duration * rate);
+const totalEarn = maartjeEarnPerHour.reduce((acc, current) => acc + current);
+console.log('€' + totalEarn.toFixed(2));
```

Comment:
the functionality of this functionality is working well, so good job.

You have a lot of code and you have used a lot of variables and functions. Instead you could have used one function.  I will provide an example below. Try to review the example below, understand every step, and update your code based on the comments and your new understanding.

const totalSalary = tasks.map(minutesToHours => minutesToHours.duration / 60)
.filter(twoAndMore => twoAndMore >= 2)
.map(perHourSalary => (perHourSalary * 16))
.reduce((sum, eachSalary) => sum += eachSalary, 0)
.toFixed(2);

Reply:
Thank you Sil for the feedback! Point taken.